### PR TITLE
Added note for beta.

### PR DESCRIPTION
**Pull request by shantanuchhabra**

### CODE REVIEW

File: src/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py [hunk 1]
```diff
@@ -21,7 +21,9 @@ def create(data,
            max_iterations=0,
            verbose=True):
     """
-    Create a :class:`OneShotObjectDetector` model.
+    Create a :class:`OneShotObjectDetector` model. Note: The One Shot Object Detector
+    is currently in beta, as we make the toolkit faster and even more performant.
```
Comment (srikris) [inline src/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py]: I don't think you need to say things like "stay tuned etc." This is currently in beta is more than enough.

File: userguide/one_shot_object_detection/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # One-Shot Object Detection
 
+*Note: The One Shot Object Detector is currently in beta, as we make the toolkit faster and even more performant. Please stay tuned for best results with the toolkit!*
```
Comment (srikris) [inline userguide/one_shot_object_detection/README.md]: Same note here.

Comment (shantanuchhabra) [reply userguide/one_shot_object_detection/README.md]: Done.

File: src/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py (hunk 1 above)
Comment (shantanuchhabra) [reply src/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py]: Done.

**Pull request merged**